README.md

```diff
@@ -71,6 +71,8 @@ docker run \
   -it \
   --rm \
   --name shell-server \
+  -p 8080:80 \
```

Comment:
is there a way to set up a hostname like docker-compose does so you can `ssh hashbang -p 22`? alternatively could we just do it ourselves with docker-compose? or is that a thing that only works inside of the docker-compose containers?